distribution/pom.xml

```
@@ -379,34 +379,74 @@
               <name>drill</name>
```

currently the name of rpm is `drill-1.0-SNAPSHOT.noarch.rpm`
```suggestion
              <name>apache-drill</name>
```
And edit an above string:
from `<version>1.0</version>` -> `<version>1.16.0-SNAPSHOT</version>`

---

OK sure I will make the rpm tobe named as apache-drill-1.16.0-SNAPSHOT.noarch.rpm
How about below code change
[centos@centos drill]$ git diff
diff --git a/distribution/pom.xml b/distribution/pom.xml
index 5005155..e73bfd7 100644
--- a/distribution/pom.xml
+++ b/distribution/pom.xml
@@ -196,9 +196,7 @@
         <inherited>true</inherited>
         <configuration>
           <excludes>
-            <exclude>**/conffiles</exclude>
             <exclude>**/git.properties</exclude>
-            <exclude>**/control</exclude>
             <exclude>**/*.checkstyle</exclude>
             <exclude>**/*.json</exclude>
             <exclude>**/README.md</exclude>
@@ -371,12 +369,12 @@
               </execution>
             </executions>
             <configuration>
-              <copyright>2013 ASF</copyright>
+              <license>http://www.apache.org/licenses/LICENSE-2.0</license>
               <group>Apache Software Foundation</group>
               <prefix>/opt</prefix>
               <release>SNAPSHOT</release>
-              <version>1.0</version>
-              <name>drill</name>
+              <version>1.16.0</version>
+              <name>apache-drill</name>
               <mappings>
                 <mapping>
                   <directory>/opt/drill</directory>
[centos@centos drill]$

---

@vdiravka That's really a very good review.  Thank you very much for suggestions.  Let me know if there are any more changes. Not sure keeping as SNAPSHOT makes sense after mentioning a release version.  I will also go through and double check any more changes are required.